.github/workflows/ci-build.yml

```diff
@@ -111,4 +114,8 @@ jobs:
     - name: Build
       run: |
         cmake -Bbuild -DOGRE_BUILD_DEPENDENCIES=OFF -DSWIG_EXECUTABLE=none -DOGRE_CONFIG_DOUBLE=ON -DOGRE_ASSERT_MODE=1 -DOGRE_PROFILING=ON .
-        cmake --build build -- -j 4
\ No newline at end of file
+        cmake --build build -- -j 4
+    - name: Build with wayland
```

Comment:
just add `-DOGRE_GLSUPPORT_USE_WAYLAND=ON` to the block above instead of introducing a new build just for this

Reply:
Done.